src/Uno.UI/Uno.UI.netcoremobile.csproj

```diff
@@ -136,6 +137,17 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
Are these changes intended?

Reply:
Nope, they weren't! 👀 Thanks for pointing it out, I will fix when I get back on this, there are still a few things that need to be adjusted before it is ready. (not a priority currently, just wanted to make a draft PR for transparency and possible feedback like this)